src/runner/PlutoRunner.jl

```diff
@@ -512,13 +514,13 @@ function run_expression(
     # We add the time it took to macroexpand to the time for the first call,
     # but we make sure we don't mention it on subsequent calls
     expansion_runtime = if expanded_cache.did_mention_expansion_time === false
-        # Is this really the easiest way to clone a struct with some changes? Pfffft
+        did_mention_expansion_time = false
```

Comment:
```suggestion
        did_mention_expansion_time = true
```

Reply:
Awesome!!! :heart:  I just wanted to go on a long debugging quest to find the cause for the failure!